### PR TITLE
Added AM records to prod.yml, sandbox.yml and playbook.yml

### PR DESCRIPTION
**Pull request by zielinskikamil**
We are starting a new project - Access Management (AM)
Added entries to prod, sandbox and playbook in order preview AM project in Jenkins

### CODE REVIEW

File: env_vars/sandbox.yml [hunk 1]
```diff
@@ -287,6 +287,24 @@ organization_folders:
...
     regex_SCM_source_filter: '(fpl-.*)'
+  - name: 'HMCTS_Sandbox_AM'
+    repo_owner: 'HMCTS'
+    displayName: 'HMCTS - AM'
+    description: 'HMCTS public repositories AM'
+    credentials_id: 'jenkins-github-hmcts-api-token_am'
+    github_api_url: 'https://api.github.com'
+    jenkinsfile_path: ['Jenkinsfile_parameterized', 'Jenkinsfile']
+    branches_to_include: '*'
+    regex_SCM_source_filter: '(snl.*)'
```
Comment (r-kruk-kainos) [inline env_vars/sandbox.yml]: Please change to am

File: env_vars/prod.yml [hunk 2]
```diff
@@ -302,6 +302,25 @@ organization_folders:
     jenkinsfile_path: ['Jenkinsfile_nightly']
     branches_to_include: 'master'
     regex_SCM_source_filter: '(fpl-.*)'
+  - name: 'HMCTS_AM'
+    repo_owner: 'HMCTS'
+    displayName: 'HMCTS - AM'
+    description: 'HMCTS public repositories AM'
+    credentials_id: 'jenkins-github-hmcts-api-token_am'
```
[the remark this answers is not in the document]
Comment (zielinskikamil) [reply env_vars/prod.yml]: It needs to be created, any ideas how to do it?

[the remark this answers is not in the document]
Comment (zielinskikamil) [reply env_vars/prod.yml]: isnt is included in playbook.yml ?

[the remark this answers is not in the document]
Comment (timja) [reply env_vars/prod.yml]: i can do it

[the remark this answers is not in the document]
Comment (zielinskikamil) [reply env_vars/prod.yml]: so plz do :)

[the remark this answers is not in the document]
Comment (timja) [reply env_vars/prod.yml]: done

**Pull request merged**